### ISSUE TITLE
Script doesn't work anymore

### ISSUE THREAD

**Issue by efleurant** [peppy/mixcloud-downloader#4]
Errors out with : No server found for download

**Comment by ivansabik** [comment]
Just tried it, also working for me

**Comment by penibelst** [comment]
It doesn’t work since one week.

**Comment by peppy** [comment]
so fix it? >_>

**Comment by ivansabik** [comment]
Should work, I've been working with a fork of mine which is almost a copy of the original but for CLI instead of web (https://github.com/ivansabik/mixcloud-downloader).
You could also further describe what "It doesn’t work since one week." means

**Comment by peppy** [comment]
I'm pretty sure I used it within the last week without issue, too.

**Comment by penibelst** [comment]
This [mix](https://www.mixcloud.com/patato-tunnes/black-sauce-vol54/) worked on 2015-05-25. If now I paste the link and press enter, the progress bar moves, but no link to the audio appears below.

**Comment by penibelst** [comment]
I tried random mixes, they don’t work here. I’m using Firefox 38.

**Comment by penibelst** [comment]
The latest one I downloaded with the script was on 2015-06-09.

**Comment by ivansabik** [comment]
You should continue digging into the issue anatol but the core script which obtains a downloadable URL(https://stream13.mixcloud.com/c/m4a/64/9/4/c/9/3511-9f10-48ad-ab3b-e38b3d4b8806.m4a) is working:

```bash
$ ./mixcloud https://www.mixcloud.com/patato-tunnes/black-sauce-vol54/
File name: black-sauce-vol54.m4a
Download URL: https://stream13.mixcloud.com/c/m4a/64/9/4/c/9/3511-9f10-48ad-ab3b-e38b3d4b8806.m4a
Downloading, please wait
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0 29.7M    0  271k    0     0  14887      0  0:34:54  0:00:18  0:34:36     0
```

**Comment by penibelst** [comment]
Hmm, maybe the [live demo](http://bm4.ppy.sh/mixcloud.php) is blocked? That is what I use.

**Comment by peppy** [comment]
live demo is probably not working on hhvm (switched that server across recently). please deploy it yourself (you can use `php -S 127.0.0.1:8080` if you don't have a server setup).

**Comment by penibelst** [comment]
@peppy Thanks. The script works on my machine (PHP 5.5.12-2ubuntu4.4). When #8 is merged, users like me could report issues in a better way.

**Comment by peppy** [comment]
Fixed by 160101498893704157b1ed809eb9c967477e36c2.
